src/styles/_custom_variables.scss

```diff
@@ -24,14 +24,10 @@
   --ds-header-logo-height-xs: 50px;
   --ds-header-icon-color: #{$link-color};
   --ds-header-icon-color-hover: #{$link-hover-color};
-  --ds-navbar-bg: var(--ds-header-bg);
   --ds-header-navbar-border-top-color: #{$white};
   --ds-header-navbar-border-bottom-color: #{$gray-400};
   --ds-navbar-link-color: #{$cyan};
   --ds-navbar-link-color-hover: #{darken($cyan, 15%)};
-  --ds-expandable-navbar-bg: var(--ds-navbar-bg);
-  --ds-expandable-navbar-link-color: var(--ds-navbar-link-color);
-  --ds-expandable-navbar-link-color-hover: var(--ds-navbar-link-color-hover);
```

Comment:
This appears to be accidently removing CSS styles which are used in DSpace.  Why are all these styles being removed?

Reply:
My apologies - I have restored the removed styles. I was working with another coworker on this PR and it seems like there was some issue with merging of our work. It has been resolved now.